doc/allclasses-frame.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!-- NewPage -->
+<html lang="en">
+<head>
+<!-- Generated by javadoc (1.8.0_181) on Thu Sep 06 14:54:08 EDT 2018 -->
```

Comment:
The javadoc should not be part of the source code, it can be generated with maven any time. How comes that you have it in `./doc/`? Maven should put it into `./target/site/apidocs/` (and `/target/` is in `.gitignore` and therefore ignored).

Reply:
Ah, this is my bad completely. I was playing around in Eclipse and generated that. I will remove the folder.